helm/templates/deployment.yaml

```diff
@@ -0,0 +1,118 @@
+{{- $root := . -}}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "ucentralsec.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "ucentralsec.name" . }}
+    helm.sh/chart: {{ include "ucentralsec.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  strategy:
+    type: {{ .Values.strategyType }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "ucentralsec.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      {{- with .Values.services.ucentralsec.labels }}
+      {{- toYaml . | nindent 6 }}
+      {{- end }}
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include "ucentralsec.config" . | sha256sum }}
+      labels:
+        app.kubernetes.io/name: {{ include "ucentralsec.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        {{- with .Values.services.ucentralsec.labels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+
+      containers:
+
+        - name: ucentralsec
+          image: "{{ .Values.images.ucentralsec.repository }}:{{ .Values.images.ucentralsec.tag }}"
+          imagePullPolicy: {{ .Values.images.ucentralsec.pullPolicy }}
+
+          command:
+            - /ucentral/ucentralsec
+            - --file=/ucentral-data/ucentralsec.properties
```

Comment:
Has to be changed to `/ucentralsec-data/ucentralsec.properties` at this point. But do we need to append this option? Should be the default anyway as far as I know.

Reply:
Yes, good notice

Reply:
Fixed, also fixed in ucentralsec